CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # How to Contribute
 
-The Helm Operator is [Apache 2.0 licensed](LICENSE) and accepts contributions
+The Flux Helm Operator is [Apache 2.0 licensed](LICENSE) and accepts contributions
```

Comment:
```suggestion
The Helm Operator is [Apache 2.0 licensed](LICENSE) and accepts contributions
```

We only call it 'Flux Helm Operator' when additional context is required, given someone is already 'in' the Flux ecosystem I would simply omit the project branding here.